code/modules/religion/rites.dm

```diff
@@ -109,6 +109,30 @@
 	human2borg.visible_message("<span class='notice'>[human2borg] has been converted by the rite of [name]!</span>")
 	return TRUE
 
+/datum/religion_rites/upgrade_blessing
+	name = "Receive Blessing"
+	desc = "Receive a blessing from the machine god to further your ascension."
+	ritual_length = 15 SECONDS
+	ritual_invocations = list("With hammer and faith our bodies were forged",
+								"Oh,Machine-god,We are your instruments of faith",
```

Comment:
```suggestion
								"Oh, Machine-god, We are your instruments of faith",
```
the capitalization and/or punctuation here might also have to be fixed, I'm just fixing the spacing